CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Added
+- `call-runtime` support
```

Comment:
```suggestion
-  Add `call-runtime` support - [#1641](https://github.com/paritytech/ink/pull/1641)
```

Reply:
Oops sorry for applying this suggestion - wrong window, I though this was my PR :see_no_evil: